docker/nginx/conf.d/include/proxy-cache-downloads

```diff
@@ -4,18 +4,18 @@ proxy_cache_min_uses 3; # cache after 3 uses
 proxy_cache_valid 200 206 307 308 48h; # keep 200, 206, 307 and 308 responses valid for up to 2 days
 add_header X-Proxy-Cache $upstream_cache_status; # add response header to indicate cache hits and misses
 
+# map skyd env variable value to "1" for true and "0" for false (expected by proxy_no_cache)
+set_by_lua_block $skyd_disk_cache_enabled {
+    return os.getenv("SKYD_DISK_CACHE_ENABLED") == "true" and "1" or "0"
+}
+
 # bypass - this will bypass cache hit on request (status BYPASS)
 # but still stores file in cache if cache conditions are met
-proxy_cache_bypass $cookie_nocache $arg_nocache;
+proxy_cache_bypass $cookie_nocache $arg_nocache $skyd_disk_cache_enabled;
```

Comment:
adding `$skyd_disk_cache_enabled` check to `proxy_cache_bypass` in addition to `proxy_no_cache` will cause nginx to report cache status `"BYPASS"` instead of `"MISS"` which makes more sense